sbomtool/cmd/sbomtool/root_test.go

```diff
@@ -69,7 +69,7 @@ func TestPersistentFlags(t *testing.T) {
 		// Then: Flag should be registered with correct defaults
 		require.NotNil(t, flag, "log-level flag should be registered")
 		assert.Equal(t, "info", flag.DefValue, "Default log level should be 'info'")
-		assert.Equal(t, "Log level (debug, info, warn, error)", flag.Usage, "Flag usage should be descriptive")
+		assert.Equal(t, "Set logging verbosity (debug=verbose, info=standard, warn=important, error=critical)", flag.Usage, "Flag usage should be descriptive")
```

Comment:
I wouldn't bother about these type of unit tests. They will be a pain to maintain and don't actually prevent any unwanted behavior.